docs/registry/entities/service.md

```diff
@@ -11,17 +11,27 @@
 
 **type:** `service`
 
-**Description:** A service instance.
+**Description:** A logical unit of an application or system that performs a specific function.
 
 | Role | Key | Stability | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Value Type | Description | Example Values |
 |---|---|---|---|---|---|---|
 | Identity | [`service.name`](/docs/registry/attributes/service.md) | ![Stable](https://img.shields.io/badge/-stable-lightgreen) | `Required` | string | Logical name of the service. [1] | `shoppingcart` |
-| Identity | [`service.instance.id`](/docs/registry/attributes/service.md) | ![Development](https://img.shields.io/badge/-development-blue) | `Recommended` | string | The string ID of the service instance. [2] | `627cc493-f310-47de-96bd-71410b7dec09` |
-| Identity | [`service.namespace`](/docs/registry/attributes/service.md) | ![Development](https://img.shields.io/badge/-development-blue) | `Recommended` | string | A namespace for `service.name`. [3] | `Shop` |
```

Comment:
is it ok that the Identity for `service` is only unique within another Entity (`service.namespace`)?

it wasn't clear to me from reading https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/entities/data-model.md#minimally-sufficient-identity